### PR TITLE
FB - Migrate Xara website to OVHcloud

### PR DESCRIPTION
**Pull request by fabienOVH**

### CODE REVIEW

**Comment by fabienOVH** [comment]
Waiting for the "GO" of Guillaume F. in this conv

**Comment by GuiF008** [comment]
All good for me. Go

File: pages/web_cloud/web_hosting/migrate_xara_website_to_ovh/guide.fr-fr.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+## Prérequis
+
+- Être connecté à l'interface d'administration de Xara
+
+## En pratique
+
+### Étape 1 : sauvegarder les fichiers et la base de données de votre site web Xara
+
+La première étape consiste à récupérer tous les fichiers relatifs à votre site web Xara. Cela inclut les fichiers Xara, ainsi que votre base de données si vous en avez une. Pour plus de détail, consultez la page [Exporting a Website](https://webdesigner.xara.com/bhavtest/test1/xara_desktop/product_support/web_features/exporting_website.html?rhhlterm=website){.external} ou encore l'étape 3 du guide « [Migrer son site web et ses services associés vers OVHcloud](/pages/web_cloud/web_hosting/hosting_migrating_to_ovh) ».
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_xara_website_to_ovh/guide.fr-fr.md]:
```suggestion
La première étape consiste à récupérer tous les fichiers relatifs à votre site web Xara. Cela inclut les fichiers Xara, ainsi que votre base de données si vous en avez une. Pour plus de détails, consultez la page « [Exporting a Website](https://webdesigner.xara.com/bhavtest/test1/xara_desktop/product_support/web_features/exporting_website.html?rhhlterm=website){.external} » ou encore l'étape 3 du guide « [Migrer son site web et ses services associés vers OVHcloud](/pages/web_cloud/web_hosting/hosting_migrating_to_ovh) ».
```

File: pages/web_cloud/web_hosting/migrate_xara_website_to_ovh/guide.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+Une fois les fichiers et la base de données de votre site web Xara sauvegardés, transférez-les vers votre hébergement web OVHcloud. Si vous ne possédez pas encore d'hébergement web OVHcloud, suivez l'étape 1 du guide « [Migrer son site web et ses services associés vers OVHcloud](/pages/web_cloud/web_hosting/hosting_migrating_to_ovh) ».
+
+#### Étape 2.1 : transférer les fichiers de votre site web Xara
+
+> [!primary]
+>
+> Nous vous recommandons d'utiliser le logiciel [FileZilla](/pages/web_cloud/web_hosting/ftp_filezilla_user_guide) pour transférer vos fichiers Xara vers votre hébergement web.
+>
+
+Pour transférer les fichiers relatifs à votre site web Xara, connectez-vous d'abord à [l'espace de stockage FTP de votre hébergement web OVHcloud](/pages/web_cloud/web_hosting/ftp_connection).
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_xara_website_to_ovh/guide.fr-fr.md]:
```suggestion
Pour transférer les fichiers relatifs à votre site web Xara, connectez-vous  à l'[espace de stockage FTP de votre hébergement web OVHcloud](/pages/web_cloud/web_hosting/ftp_connection).
```

File: pages/web_cloud/web_hosting/migrate_xara_website_to_ovh/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Une fois connecté à l'espace de stockage FTP de votre hébergement web OVHcloud, naviguez vers le répertoire racine « www » (ou dans un autre dossier racine que vous aurez préalablement créé). Si vos fichiers de sauvegarde sont compressés (zippés), décompressez-les dans un dossier vide sur votre ordinateur avant de les téléverser dans le répertoire racine de votre hébergement web OVHcloud.
+
+#### Étape 2.2 : importer la base de données de votre site web Xara
+
+Si vous n'en avez pas encore, [créez une nouvelle base de données](/pages/web_cloud/web_hosting/sql_create_database) puis [importez votre sauvegarde dans votre nouvelle base de données](/pages/web_cloud/web_hosting/sql_importing_mysql_database).
+
+> [!primary]
+>
+> OVHcloud propose des serveurs de base de données Web Cloud Databases. Si vous souhaitez utiliser cette offre avec votre site web, retrouvez l'ensemble de notre documentation sur ce produit sur notre [page dédiée](/links/web/databases).
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_xara_website_to_ovh/guide.fr-fr.md]:
```suggestion
> OVHcloud propose des serveurs de base de données Web Cloud Databases. Si vous souhaitez utiliser cette offre avec votre site web, retrouvez l'ensemble des informations sur ce produit sur notre [page dédiée](/links/web/databases).
```

File: pages/web_cloud/web_hosting/migrate_xara_website_to_ovh/guide.fr-fr.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+### Migrer d'autres services associés à votre site web Xara
+
+Vous venez de migrer vos fichiers et votre base de données Xara. Si vous souhaitez migrer d'autres services comme vos e-mails, votre nom de domaine ou encore vos zones DNS, suivez les étapes du guide « [Migrer son site web et ses services associés vers OVHcloud](/pages/web_cloud/web_hosting/hosting_migrating_to_ovh) » en vous reportant aux étapes qui concernent les services que vous souhaitez migrer. En effet, plusieurs des étapes auront déjà été effectuées dans ce guide.
+
+> [!warning]
+>
+> Si vous souhaitez migrer votre nom de domaine Xara vers OVHcloud, consultez la partie 3 de cette [page du site officiel de Xara](https://www.xara-online.com/fr/domaines-faq.152.html){.external} pour obtenir des informations détaillées sur les étapes administratives nécessaires, comme le déblocage de votre nom de domaine et l'obtention du code de transfert.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_xara_website_to_ovh/guide.fr-fr.md]:
```suggestion
> Si vous souhaitez migrer votre nom de domaine Xara vers OVHcloud, consultez la partie 3 de cette [page du site officiel de Xara](https://www.xara-online.com/fr/domaines-faq.152.html){.external}. Vous-y trouverez des informations détaillées sur les étapes administratives nécessaires, comme le déblocage de votre nom de domaine et l'obtention du code de transfert.
```

**Review by Kazarus4 — COMMENTED** [review]
First review done

**Comment by fabienOVH** [comment]
Review taken into account. Ok for me

File: pages/web_cloud/web_hosting/migrate_xara_website_to_ovh/guide.fr-fr.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
+---
+title: "Migrer son site web Xara vers OVHcloud"
+excerpt: "Découvrez comment migrer votre site web Xara et ses services associés vers OVHcloud"
+updated: 2024-07-24
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_xara_website_to_ovh/guide.fr-fr.md]:
```suggestion
updated: 2024-07-29
```

**Comment by Kazarus4** [comment]
We must prod before the 1sr of August 2024. @Y0Coss / @OVH-Team-Guides on holidays, so I checked FRFR and I will merge this PR . This guide is very similar to another guide. That's why I'll do the translations. OVH-Team-Guides / Y0Coss will do some proof after his return from Holidays last week.

**Pull request merged**